### DISCUSSION TITLE
Upload an image through API

### DISCUSSION THREAD

**Discussion by zBilalz** [civitai/civitai#1218] [Q&A]
Is it possible to make a request to the API to upload an image to your account?

**Comment by Sadafshaik-786** [comment]
Hi @zBilalz,
Right now the public CivitAI API is read-only — it doesn’t support uploading images or models to your account via API. Uploads can only be done through the web UI. The team has mentioned they’re considering expanding the API in the future, but there’s no ETA yet.
